### PR TITLE
Set `false` as default value for fullText in hugging face config

### PR DESCRIPTION
**Pull request by GregJohnStewart**
Fixed issue returning the full text by default for hugging face

### CODE REVIEW

File: model-providers/hugging-face/runtime/src/main/java/io/quarkiverse/langchain4j/huggingface/runtime/config/ChatModelConfig.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ public interface ChatModelConfig {
     /**
      * If set to {@code false}, the return results will not contain the original query making it easier for prompting
      */
+    @WithDefault("false")
     Optional<Boolean> returnFullText();
```
Comment (gastaldi) [inline model-providers/hugging-face/runtime/src/main/java/io/quarkiverse/langchain4j/huggingface/runtime/config/ChatModelConfig.java:48]: If a default value is specified, it's a good idea to remove the `Optional`

```suggestion
    boolean returnFullText();
```

Comment (gastaldi) [reply model-providers/hugging-face/runtime/src/main/java/io/quarkiverse/langchain4j/huggingface/runtime/config/ChatModelConfig.java:48]: Also, a better PR title would make it look great in the release notes 😉

**Comment by GregJohnStewart** [comment]
I'll update that in a few

**Comment by GregJohnStewart** [comment]
@gastaldi 

Okay, both title and optional adjusted

**Comment by GregJohnStewart** [comment]
Hold on, fixing a usage that expected an optional

**Comment by GregJohnStewart** [comment]
Okay, fixed and up for review. The tests for this take _forever_ lol

**Review by geoand — APPROVED** [review]
Thanks

**Pull request merged**